test/scripts/google_test.js

```diff
@@ -551,4 +551,69 @@ describe('google', () => {
 
     sharedExamplesForYoutubeError('!brownbagnext', [{ max: 1, status: 'upcoming' }])
   })
+  describe('!brownbaglast', () => {
```

Comment:
use blank lines to separate code blocks

```suggestion

  describe('!brownbaglast', () => {
```